package.json

```diff
@@ -83,13 +85,13 @@
     "react-full-screen": "^1.1.0",
     "react-helmet": "^6.1.0",
     "react-hot-toast": "^2.2.0",
+    "react-instantsearch": "^7.5.0",
     "react-instantsearch-dom": "^6.28.0",

```

Comment:
why are these version numbers different? (do we still need `react-instantsearch-dom`?)

Reply:
oh oops i forgot to remove it 😅